doc/netplan.md

```diff
@@ -576,6 +576,12 @@ Example:
       enp1s16f1:
         link: enp1
 
+``virtual-functions`` (scalar)
+
+:    (SR-IOV devices only) Explicit number of Virtual Functions for the given
+     Physical Function. This is optional, as normally netplan will allocate as
+     many VFs as there are needed.
```

Comment:
I think we should state more explicitly that this is only for special/advanced configurations, e.g. something like:

"Additional VFs can be configured outside of netplan. This should be used for special cases only."

Reply:
+1, let me adjust this a bit better, as proposed.